src/PrestaShopBundle/Controller/Admin/Sell/Catalog/FeatureController.php

```diff
@@ -26,163 +26,1921 @@
+        }
+        $this->getContext()->smarty->assign('help_link', 'https://help.prestashop.com/' . Language::getIsoById($this->getUser()->getData()->id_lang) . '/doc/'
+            . Tools::getValue('controller') . '?version=' . _PS_VERSION_ . '&country=' . Language::getIsoById($this->getUser()->getData()->id_lang));
+    }
+
+    /**
+     * Set default toolbarTitle to admin breadcrumb.
+     */
+    public function initToolbarTitle()
+    {
+        $this->toolbarTitle = array_unique($this->breadcrumbs);
+
+        //switch ($this->display) {
+        //    case 'edit':
+        //        $this->toolbarTitle[] = $this->trans('Edit');
+        //        $this->addMetaTitle($this->trans('Edit'));
+        //
+        //        break;
+        //
+        //    case 'add':
+        //        $this->toolbarTitle[] = $this->trans('Add new');
+        //        $this->addMetaTitle($this->trans('Add new'));
+        //
+        //        break;
+        //
+        //    case 'view':
+        //        $this->toolbarTitle[] = $this->trans('View');
+        //        $this->addMetaTitle($this->trans('View'));
+        //
+        //        break;
+        //}
+
+        if ($filter = $this->addFiltersToBreadcrumbs()) {
+            $this->toolbarTitle[] = $filter;
+        }
+    }
+
+    /**
+     * @return string|void
+     */
+    public function addFiltersToBreadcrumbs()
+    {
+        if ($this->filter) {
+            $filters = [];
+
+            foreach ($this->fieldsList as $field => $t) {
+                if (isset($t['filter_key'])) {
+                    $field = $t['filter_key'];
+                }
+
+                if (($val = Tools::getValue($this->table . 'Filter_' . $field)) || $val = $this->getContext()->cookie->{$this->getCookieFilterPrefix() . $this->table . 'Filter_' . $field}) {
+                    if (!is_array($val)) {
+                        $filter_value = '';
+                        if (isset($t['type']) && $t['type'] == 'bool') {
+                            $filter_value = ((bool) $val) ? $this->trans('yes', 'Admin.Actions') : $this->trans('no', 'Admin.Actions');
+                        } elseif (isset($t['type']) && $t['type'] == 'date' || isset($t['type']) && $t['type'] == 'datetime') {
+                            $date = json_decode($val, true);
+                            if (isset($date[0])) {
+                                $filter_value = $date[0];
+                                if (isset($date[1]) && !empty($date[1])) {
+                                    $filter_value .= ' - ' . $date[1];
+                                }
+                            }
+                        } elseif (is_string($val)) {
+                            $filter_value = htmlspecialchars($val, ENT_QUOTES, 'UTF-8');
+                        }
+                        if (!empty($filter_value)) {
+                            $filters[] = $this->trans('%s: %s', 'Admin.Actions', [$t['title'], $filter_value]);
```

Comment:
Could you give me more context for this string, please? Are you sure this is an action?